### DISCUSSION TITLE
How to config container background

### DISCUSSION THREAD

**Discussion by FeiPengDev** [vueform/builder#86] [Questions] [answered]
Excuse me, I noticed that the **container's background color** is visible during the build process, but if I want to configure the container's background color during rendering as well, is there a solution available for this? 
**Thank you very much for answering my questions!**
![image](https://github.com/user-attachments/assets/882f899d-a539-493c-a4ad-6665468567c6)

**Comment by adamberecz** [comment, marked as answer]
You can achieve something similar by adding classes to the Group/Object element in the renderer's `vueform.config.js`:

```js
// vueform.config.js

import { defineConfig } from '@vueform/vueform'

export default {
  // ...
  addClasses: {
    GroupElement: {
      container: 'bg-gray-200 bg-opacity-50 p-3 rounded',
    },
    ObjectElement: {
      container: 'bg-gray-200 bg-opacity-50 p-3 rounded',
    },
  },
};
```

Demo:
https://stackblitz.com/edit/github-svqwqg?file=src%2FApp.vue,vueform.config.js

If this is not an option, because the renderer and the builder are in the same project, you can use [presets](https://vueform.com/docs/styles-and-layout#presets) on rendered forms.
